### ISSUE TITLE
SDK should log/throw native available error only once on start

### ISSUE THREAD

**Issue by krystofwoldrich** [getsentry/sentry-react-native#3028]
### Description

The constant logging/throwing of the native not available error can lead to endless breadcrumbs.

**Comment by krystofwoldrich** [comment]
Duplicate of https://github.com/getsentry/sentry-react-native/issues/3027
